NEWS.md

```diff
@@ -1,6 +1,15 @@
+## 3.0.0 in progress
+### Breaking changes
+* Update module to Java 17 & the latest dependencies ([FHIQC-30](https://issues.folio.org/browse/FHIQC-30))
+
+### Dependencies
+* Bump `java` from `11` to `17`
+* Bump `log4j` from `2.19.0` to `2.20.0`
+* Bump `vertx` from `4.3.8` to `4.4.0`
+
 ## 2.3.0 2023-02-14
 ### Dependencies
-* Bump `folio-service-tools` from `1.10.0` to `1.10.1`
+* Bump `folio-service-tools` from `1.10.0` to `3.1.0-SNAPSHOT`
```

Comment:
You should not change news for released version.
Version bump from 1.10.1 to 3.1.0 should be in 3.0.0 version section